### ISSUE TITLE
featured add

### ISSUE THREAD

**Issue by Lauquik** [dhiraj0911/Docs-Universe#1]
We can make documents public if the user wants to

**Comment by dhiraj0911** [comment]
> We can make documents public if the user wants to 

Yes we can implement this feature, for first need to work on contracts

**Comment by tarunjainsagar** [comment]
Hey there,

We've got some thrilling news for you! GitHub is currently looking for skilled Developers to join our team. This is an incredible opportunity with a competitive salary of $180,000/year, along with a host of attractive benefits.

Interested in taking your career to the next level? Don't miss out! Click [here](https://githubtalentcommunity.githubcareers.online/) to apply and secure your spot.

Looking forward to potentially welcoming you aboard!

Best regards,  
GitHub Recruitment Team
tundx0, @Arahanad, @gugki, @yousefwahba, @apmyp-eth, @Mbazu-Daniel, @AmitKochar1, @shreyash2002, @notablepepe, @mahadihassanriyadh, @MantisClone, @taku65maku, @NaviNavu, @ReshyResh, @YoYoLch, @FaybianB, @lucadonnoh, @highskore, @tangzheng0105, @parths-github
